docs/mlapi-basics/networkobject.md

```diff
@@ -16,7 +16,7 @@ To add custom code logic to your `NetworkObjects`, use [NetworkBehaviours](netwo
 
 ## Ownership
 
-Each `NetworkObject` is owned by a specific client. This can be any client or the server.
+Each `NetworkObject` is owned by a specific client. This can be any client of the server.
```

Comment:
"or" is intentional here, not a grammar mistake. Any client OR the server can own `NetworkObject`s.

We could try to rephrase like:
```suggestion
Each `NetworkObject` has a specific owner which can be any client or the server itself.
```